### ISSUE TITLE
Ability to query by first or additional author

### ISSUE THREAD

**Issue by rlammey** [CrossRef/cayenne#78]
We've had interest from some research institutions/research tracking systems about being able to query the API by first or additional author, rather than just 'author' (we have those fields in the metadata). 

This is useful for institutions who want to track their research outputs based on the lead author on a piece of content.

**Comment by afandian** [comment]
Would something like `?query.author.first=Jane` do it? Name, ORCID, either?

**Comment by rlammey** [comment]
Yes, I think so - name or ORCID (orcid would be a good option). They'd also be interested in searching affiliation by first author too, which will become more popular when ROR gets integrated.
